src/server/helpers/middleware.ts

```diff
@@ -41,11 +41,9 @@ function makeLoader(modelName, propName, sortFunc?, relations = []) {
 			const resultsSerial = results.toJSON();
 			res.locals[propName] =
 				sortFunc ? resultsSerial.sort(sortFunc) : resultsSerial;
-			
-			
-			}
+		}
 		catch (err) {
-			next(err);
+			return next(err);
```

Comment:
Usually I also use the `return next()` pattern in express to save one line of code.
This works because the return value of a middleware is discarded. (The return type of `next()` is `void`, so it could be anything and if the type did matter, even the previous way to return `null` could cause unexpected behaviour in the previous middleware which is calling ours via `next`.)
See also https://stackoverflow.com/questions/16810449/when-to-use-next-and-return-next-in-node-js

Reply:
Shall I leave it as it is, then?

Reply:
Thanks for the research @kellnerd !

@stoichy Yes, you can leave it as-is :)